web/src/pages/next-seo.config.ts

```diff
@@ -14,7 +14,8 @@ export const getDefaultSeoConfig = (pathname: string): NextSeoProps => ({
 		type: "website",
 		locale: "en_GB",
 		url: publicRuntimeConfig.baseURL + pathname,
-		site_name: "NICE",
+		site_name:
+			"NICE website: the National Institute for Health and Care Excellence",
```

Comment:
should 'the' be capitalised as the whole name of the organisation is a proper noun?